index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
+              <dd>To request or specify <a data-lt="expanded document form">expanded JSON-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+              <dd>To request or specify <a data-lt="compacted document form">compacted JSON-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#context</code></dt>
+              <dd>To request or specify a <a data-lt="context document">JSON-LD context document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+              <dd>To request or specify <a data-lt="flattened document form">flattened JSON-LD document form</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#frame</code></dt>
+              <dd>To request or specify a <a data-lt="frame">JSON-LD frame document</a>.</dd>
+              <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+              <dd>To request or specify <a data-lt="framed document form">framed JSON-LD document form</a>.</dd>
+            </dl>
+            <p>All other URIs starting with <code>http://www.w3.org/ns/json-ld</code>
+              are reserved for future use by JSON-LD specifications.</p>
+            <!--p>Other specifications MAY create further structured subtypes
+              by using `+ld+json` as a suffix for a new base subtype, as in
+              `application/example+ld+json`.
+              Unless defined otherwise, such subtypes use the same
+              fragment identifier behavior as `application/ld+json`.</p-->
+            <p>Other specifications may publish additional `profile` parameter
+              URIs with their own defined semantics.
+              This includes the ability to associate a file extension with a `profile` parameter.</p>
+            <p>
+              When used as a <a data-cite="RFC4288#section-4.3">media type parameter</a> [[RFC4288]]
+              in an <a data-cite="rfc7231#rfc.section.5.3.2">HTTP Accept header</a> [[RFC7231]],
+              the value of the <code>profile</code> parameter MUST be enclosed in quotes (<code>"</code>) if it contains
+              special characters such as whitespace, which is required when multiple profile URIs are combined.</p>
+            <p>When processing the "profile" media type parameter, it is important to
+              note that its value contains one or more URIs and not IRIs. In some cases
+              it might therefore be necessary to convert between IRIs and URIs as specified in
+              <a data-cite="RFC3986#section-5.1">section 3 Relationship between IRIs and URIs</a>
+              of [[RFC3987]].</p>
+          </dd>
+        </dl>
+      </dd>
+      <dt>Encoding considerations:</dt>
+      <dd>See <a data-cite="RFC8259#section-11">RFC&nbsp;8259, section 11</a>.</dd>
+      <dt id="iana-security">Security considerations:</dt>
+      <dd>See <a data-cite="RFC8259#section-12">RFC&nbsp;8259, section 12</a> [[RFC8259]]
```

Comment:
The preferred practice is to have a "Security Considerations" section and reference this from the media type registration. We do have such a section, but it references this. We should basically move this content to the existing "Security Considerations" section and replace this with a reference back using "See <a href="#security"></a>".